### PR TITLE
feat: transform html added

### PR DESCRIPTION
**Pull request by S4n60w3n**
Maven license text are sometimes put as html pages. This will increase analyze percentage

### CODE REVIEW

File: license_sh/analyze/analyze_shared.py [hunk 1]
```diff
@@ -131,3 +133,34 @@ def add_analyze_to_dep_tree(analyze_dict: Dict, dep_tree: AnyNode):
                 node_analyze.pop("file", None)
                 node.analyze.append(node_analyze)
     return dep_tree
+
+
+def transformHtml(htmlText: str, ignored_tags: List = IGNORED_HTML_TAGS) -> str:
```
Comment (jvorcak) [inline license_sh/analyze/analyze_shared.py:138]: rename to `transform_html` See Naming styles in https://realpython.com/python-pep8/

Comment (jvorcak) [reply license_sh/analyze/analyze_shared.py:138]: The same applies to `html_text`

Comment (S4n60w3n) [reply license_sh/analyze/analyze_shared.py:138]: Ah, javascript side is too strong with me :D

Comment (S4n60w3n) [reply license_sh/analyze/analyze_shared.py:138]: Fixed

File: tests/analyze/test_analyze_shared.py [hunk 2]
```diff
@@ -184,6 +185,47 @@ def test_get_node_analyze_dict_duplicities(
             len(result.get(get_node_id(project_name, project_version))), 1,
         )
 
+    def test_transformHtml_string(self):
```
Comment (jvorcak) [inline tests/analyze/test_analyze_shared.py:188]:
```suggestion
    def test_transform_html_string(self):
```

File: tests/analyze/test_analyze_shared.py [hunk 3]
```diff
@@ -184,6 +185,47 @@ def test_get_node_analyze_dict_duplicities(
...
+        normal_string = "This is normal string"
+        self.assertEqual(transform_html(normal_string), normal_string)
+
+    def test_transformHtml_simple(self):
```
Comment (jvorcak) [inline tests/analyze/test_analyze_shared.py:192]:
```suggestion
    def test_transform_html_simple(self):
```

File: tests/analyze/test_analyze_shared.py [hunk 4]
```diff
@@ -184,6 +185,47 @@ def test_get_node_analyze_dict_duplicities(
...
+        normal_string = "This is normal string"
+        self.assertEqual(transform_html(f"<p>{normal_string}</p>"), normal_string)
+
+    def test_transformHtml_complex(self):
```
Comment (jvorcak) [inline tests/analyze/test_analyze_shared.py:196]:
```suggestion
    def test_transform_html_complex(self):
```

File: tests/analyze/test_analyze_shared.py [hunk 5]
```diff
@@ -184,6 +185,47 @@ def test_get_node_analyze_dict_duplicities(
...
+            <Body>
+                <h3>License title</h3>
+                <p>License text</p>
+            </Body>
+</html>"""
+        result = """License title
+License text"""
+        self.assertEqual(transform_html(html_text), result)
+
+    def test_transformHtml_xml(self):
```
Comment (jvorcak) [inline tests/analyze/test_analyze_shared.py:215]:
```suggestion
    def test_transform_html_xml(self):
```

**Pull request merged**